charts/firefly-iii-stack/Chart.yaml

```diff
@@ -1,19 +1,19 @@
 apiVersion: v2
 name: firefly-iii-stack
-version: 0.5.2
+version: 0.6.0
 description: Installs Firefly III stack (db, app, importer)
 type: application
 dependencies:
 - name: firefly-db
-  version: 0.0.4
+  version: 0.2.1
   condition: firefly-db.enabled
   repository: https://firefly-iii.github.io/kubernetes/
 - name: firefly-iii
-  version: 1.0.1
+  version: 1.1.0
   condition: firefly-iii.enabled
   repository: https://firefly-iii.github.io/kubernetes/
 - name: importer
-  version: 1.1.1
+  version: 1.1.2
```

Comment:
If you open a new PR for the version bump here, please bump this to `1.1.3`. Thank you!